src/scss/pages.scss

```diff
@@ -123,6 +123,62 @@
     }
 }
 
+#sso-page {
+    .content {
+        width: 300px;
+
+        .box {
+            margin-top: 30px;
+            margin-bottom: 30px;
+            text-align: center;
+        }
+    }
+}
+
+#change-password-page {
+    .content {
+        width: 400px;
```

Comment:
why does it need to be wider?

Reply:
Honestly - it just looks better with a bit more width in regards to the `app-callout`